src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/NoticeOfChangeController.java

```diff
@@ -74,6 +75,25 @@ public ResponseEntity<CCDCallbackResponse> updateNocRespondents(@RequestBody Cal
             return ResponseEntity.status(FORBIDDEN.value()).build();
         }
 
+        try {
```

Comment:
Are there any unit tests written for these logics here?

Reply:
What tests would you expect? There is a test for changing organisation access, and there are existing unit tests for sendNotificationOfChangeEmails

Reply:
can't see the changes being reflected in the NoticeOfChangeControllerTest.java